doc/man5/flux-config-exec.rst

```diff
@@ -0,0 +1,46 @@
+===================
+flux-config-exec(5)
+===================
+
+
+DESCRIPTION
+===========
+
+The Flux system instance **job-exec** service requires additional
+configuration via the ``exec`` table, for example to enlist the services
+of a setuid helper to launch jobs as guests.
+
+The ``exec`` table may contain the following keys:
+
+
+KEYS
+====
+
+imp
+   (optional) Set the path to the IMP (Independent Minister of Privilege)
+   helper program, as described in RFC 15, so that jobs may be launched with
+   the credentials of the guest user that submitted them.  If unset, only
+   jobs submitted by the instance owner may be executed.
+
```

Comment:
`job-exec` also supports a `job-shell` key to configure a different path for the job shell (unlikely to be necessary) and a `kill-timeout` key to configure the time between SIGTERM andSIGKILL when a job is terminated.

I'm fine updating this in a future PR.

Reply:
Ah we can get that in here.  I was working backward from the admin guide and so I missed that.

Reply:
FYI - I did 
```console
$ git grep flux_conf_unpack | grep -v test/
src/broker/boot_config.c:    if (flux_conf_unpack (cf,
src/broker/broker.c:    if (flux_conf_unpack (conf, NULL, "{s:{}}", "bootstrap") == 0) {
src/broker/brokercfg.c:    if (flux_conf_unpack (flux_get_conf (h), NULL, "o", &conf) < 0)
src/broker/overlay.c:        if (flux_conf_unpack (flux_get_conf (ov->h),
src/common/libflux/conf.c:int flux_conf_unpack (const flux_conf_t *conf,
src/common/libflux/conf.h:int flux_conf_unpack (const flux_conf_t *conf,
src/modules/connector-local/local.c:    if (flux_conf_unpack (conf,
src/modules/content-s3/content-s3.c:    if (flux_conf_unpack (conf,
src/modules/job-archive/job-archive.c:    if (flux_conf_unpack (flux_get_conf (ctx->h),
src/modules/job-exec/exec.c:    if (flux_conf_unpack (flux_get_conf (h),
src/modules/job-exec/exec.c:    if (flux_conf_unpack (flux_get_conf (h),
src/modules/job-exec/job-exec.c:    if (flux_conf_unpack (flux_get_conf (h),
src/modules/job-manager/jobtap.c:    if (flux_conf_unpack (conf,
src/modules/job-manager/journal.c:    if (flux_conf_unpack (flux_get_conf (ctx->h),
src/modules/job-manager/plugins/perilog.c:    if (flux_conf_unpack (flux_get_conf (h),
src/modules/resource/inventory.c:    if (flux_conf_unpack (flux_get_conf (h),
src/modules/resource/resource.c:    if (flux_conf_unpack (conf,
```